scripts/superdesk-desks/desks.js

```diff
@@ -216,15 +216,18 @@
                  *
                  * In case it gets called multiple times it will query only once
                  */
-                function scheduleQuery() {
+                function scheduleQuery(delay) {
```

Comment:
You can do:
```js
function scheduleQuery(delay = 5000) {
```
to obtain default parameter values, and remove the `if` statement below.